samples/bot-receive-channel-messages-withRSC/csharp/README.md

```diff
@@ -0,0 +1,97 @@
+﻿﻿﻿Using this C# sample, a bot can receive all channel messages with RSC without @mention.
+For reference please check [Receive Channel messages with RSC](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/channel-messages-with-rsc)
+
+This feature shown in this sample is currently available in Public Developer Preview only.
+
+## Key features
+
+- Showing messages based on option selected
+
+![Channel messages](ReceiveMessagesWithRSC/Images/botWithRSCFlow.png)
```

Comment:
Let's also include the screenshot of install screen which shows the additional permission.

Reply:
Added the screenshots